source/includes/administration/_reports_pricing_v2.md

```diff
@@ -246,6 +247,8 @@ Customer Report Attributes | &nbsp;
 `organizations.appliedPricing.name`<br/>*Object* | A map of short language codes to their translated name for the pricing package.
 `startDate`<br/>*string* | An ISO-8601 instant format string representing the start of the report.
 `endDate`<br/>*string* | An ISO-8601 instant format string representing the end of the report.
+`orgDeleted`<br/>*boolean* | Whether or not the organization is deleted.
+`metadata`<br/>*object* | Additional information about the deletion event of the organization (optional).
```

Comment:
the 'optional' kind of implies that this is a field and not a response attribute maybe?

Reply:
d3880b7f0c8ceb8999ecf96a3eda20e3669bc262